README.md

```diff
@@ -159,6 +137,38 @@ When you run `twilio project:add`, it stores your credentials under a project ca
 
 To add a second project after the default project, you can run `twilio project:add -p my_other_proj` (using whatever identifier you'd like in place of `my_other_proj`). Then, when you run subsequent commands, just include the `-p my_other_proj` in the command (e.g. `twilio incoming-phone-number:list -p my_other_proj`).
 
+## Plugins
+
+twilio-cli can be extended via plugins.  
+
+At this time, only two plugins exist:
+
+* [twilio-run plugin](https://github.com/twilio-labs/plugin-serverless): To streamline your Twilio Functions development workflow, [Dominik Kundel](https://github.com/dkundel) created `twilio-run`. You can use twilio-run from within twilio-cli via [the twilio-run plugin](https://github.com/twilio-labs/plugin-serverless).  
```

Comment:
Confusing to call this the `run` plugin since it's the `serverless` plugin everywhere.